deploy/playbooks/roles/postgresql/tasks/standby.yml

```diff
@@ -0,0 +1,50 @@
+- name: "standby | stop postgresql"
+  sudo: yes
+  service:
+    name: postgresql
+    state: stopped
+
+- set_fact: timestamp="{{lookup('pipe','date +%Y-%m-%d-%H-%M-%S')}}"
+
+- name: "standby | move current data directory out"
+  sudo: yes
+  command: "mv {{ postgresql_cfg_data_directory }} {{ postgresql_cfg_data_directory }}_{{ timestamp }}"
+  ignore_errors: true
+
+- name: read pseudo-random password for the postgres replication user if it exists
+  local_action: command cat {{ playbook_dir }}/replication_user_password
+  register: generated_replication_user_password
+  changed_when: false
+  when: postgres_replication_user_password is not defined
```

Comment:
isn't this always defined in ``defaults/main.yml`` though? I think you might need something different in the when clause, or perhaps use ``removes`` from the ``command`` module, http://docs.ansible.com/ansible/command_module.html